File: JAVA/armstrong.java
import java.util.Scanner;
class armstrong
{
	public static void main(String args[])
	{	
		int no,sum,temp,r;
		Scanner obj=new Scanner(System.in);
		System.out.print("Enter a number to check whether it is Armstrong or not = ");
		no=obj.nextInt();
		sum=0;
		temp=no;
		while(no>0)
		{
			r=no%10;
			sum=sum+(r*r*r);
			no=no/10;
		}
		if(temp==sum)
		System.out.println("its an armstrong number");
		else
		System.out.println("its not an armstrong number");
	}
}

File: Python/armstrong.py
class test:
    def armstrong(self):
        n = int(input("Enter the number to check Armstrong = "))
        temp = n
        i = 1
        sum = None
        while i<0:
             r = n % 10
             s = s + (r * r * r)
             sum = n / 10
             i = i + 1
        if temp== sum:
            print("arm")
a = test()
a.armstrong()
